numpy/_core/include/numpy/experimental_dtype_api.h

```diff
@@ -353,7 +354,12 @@ import_experimental_dtype_api(int version)
         return 0;
     }
 
-    PyObject *multiarray = PyImport_ImportModule("numpy.core._multiarray_umath");
+    PyObject *multiarray = PyImport_ImportModule("numpy._core._multiarray_umath");
+    if (multiarray == NULL && PyErr_ExceptionMatches(PyExc_ModuleNotFoundError)) {
+        PyErr_Clear();
+        multiarray = PyImport_ImportModule("numpy.core._multiarray_umath");
+    }
+
```

Comment:
I don't care much but I think it's OK to only import from `_core` in this header, it's experimental after all.

Reply:
Make sense - I removed fallback import.